src/Reducers/initialState.js

```diff
@@ -37,7 +37,9 @@ const initialState = {
     selectedSearchResultCopy: [],
     selectedAll: false,
     employees: [],
-    selectedEmployees: []
+    selectedEmployees: [],
+    openModal: false,
+    notification: false
```

Comment:
Same question above for this notification boolean as well. I feel that if you click the 'Confirm & Remove' button, you can just pop up the alert on that event. In other words, this is more a component-level action and less of an app-level action.

Reply:
Well the reason I am storing data & put every functionality in redux. Cuz I wanted to make separation between data layer to presentainonal layer so when an application get larger I easily make find where the function & data is. But I understand your point. But I did `SnackBar` component app level but  `Modal` needs to be in redux level cuz I dont want to make pass my props down to child of child component dont want to create props drilling issue.